recipes/haesleinhuepf-pyqode.core/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "haesleinhuepf-pyqode.core" %}
+{% set version = "2.15.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/haesleinhuepf-pyqode.core-{{ version }}.tar.gz
+  sha256: 3b8ca2623244cccc5d7cf3ab02906375a597686d670bab772049e18177db16eb
+
+build:
+  entry_points:
+    - pyqode-console = pyqode.core.tools.console:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pyqt_distutils
+    - pytest
+    - python >=3.7
+  run:
+    - future
+    - pygments
+    - python >=3.7
+
+test:
+  imports:
+    - pyqode
```

Comment:
This is a "soft" fork in the sense that it preserves the original import name. In that case we should add a `run_constrained` to prevent this one from being installed along the original project.

Is that correct @haesleinhuepf?

Reply:
That is correct!